cypress/fixtures/constants.ts

```diff
@@ -70,6 +70,59 @@ export class Constants {
         diferentReactVersionsApp1ReactBlockSubheader: 'And these are children passed into it from the legacy app',
         diferentReactVersionsParagraph: 'More react components from App2 using non-legacy React to render',
         diferentReactVersionsButtonName: 'App 2 Button',
+        comprehensiveDemoButtonTextApp5: 'bar',
```

Comment:
please recheck all added constants and check what you can merge to objects

Reply:
Fixed